Plugins/GitUIPluginInterfaces/ViewModels/GitRepoViewModel.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.Design;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Messaging;
+using GitUIPluginInterfaces;
+
+namespace Plugins.GitUIPluginInterfaces.ViewModels;
+
+public interface IGitRepoViewModel : INotifyPropertyChanged
+{
+    IEnumerable<GitRevision> Revisions { get; }
```

Comment:
💡 Remove the view model and make this a dependency property on the view

Reply:
> on *the* view

There is no such thing as a single view.
How would you resolve the reason for this PR?

>> Fixes https://github.com/gitextensions/gitextensions/pull/11273#discussion_r1361134225